src/frontend/src/components/Footer.tsx

```diff
@@ -47,7 +47,7 @@ function Links() {
 }
 
 function CopyRight() {
-    const Gitalytics = () => <Link to="https://github.com/konstantinlob/gitalytics.org" className="hover:underline">Gitalytics</Link>;
+    const Gitalytics = () => <Link to="https://github.com/Gitalytics-org/gitalytics.org" className="hover:underline">Gitalytics</Link>;
```

Comment:
```suggestion
    const Gitalytics = () => <Link to="https://github.com/Gitalytics-org" className="hover:underline">Gitalytics</Link>;
```
Das Repo wird ja wahrscheinlich nicht immer public seien